package.json

```diff
@@ -78,6 +79,7 @@
     "deepmerge": "^1.5.2",
     "global-cache": "^1.2.1",
     "hoist-non-react-statics": "^2.3.1",
-    "prop-types": "^15.6.0"
+    "prop-types": "^15.6.0",
+    "react-with-direction": "^1.1.0"
```

Comment:
let's make this both a dep and a peer dep - a peer dep so there's no conflict in case they use a different version of `react-with-direction`, and "also a dep" so npm 3+ users don't have to manually add it to package.json.